golang/testsuite/testsuite_impl/local_static_file_test/local_static_file_test_.go

```diff
@@ -44,31 +45,57 @@ func (l LocalStaticFileTest) Run(network networks.Network) error {
 		return stacktrace.Propagate(err, "An error occurred getting service '%v'", testService)
 	}
 
-	staticFileAbsFilepaths, err := serviceCtx.LoadStaticFiles(map[services.StaticFileID]bool{static_file_consts.TestStaticFileID: true})
+	staticFileAbsFilepaths, err := serviceCtx.LoadStaticFiles(map[services.StaticFileID]bool{
+		static_file_consts.TestStaticFile1ID: true,
+		static_file_consts.TestStaticFile2ID: true,
+	})
 	if err != nil {
-		return stacktrace.Propagate(err, "An error occurred loading the static file corresponding to key '%v'", static_file_consts.TestStaticFileID)
+		return stacktrace.Propagate(err, "An error occurred loading the static file corresponding to key '%v'", static_file_consts.TestStaticFile1ID)
 	}
-	testFileAbsFilepath, found := staticFileAbsFilepaths[static_file_consts.TestStaticFileID]
+	testFile1AbsFilepath, found := staticFileAbsFilepaths[static_file_consts.TestStaticFile1ID]
 	if !found {
-		return stacktrace.Propagate(err, "No filepath found for test file key '%v'; this is a bug in Kurtosis!", static_file_consts.TestStaticFileID)
+		return stacktrace.Propagate(err, "No filepath found for test file 1 key '%v'; this is a bug in Kurtosis!", static_file_consts.TestStaticFile1ID)
+	}
+	testFile2AbsFilepath, found := staticFileAbsFilepaths[static_file_consts.TestStaticFile2ID]
+	if !found {
+		return stacktrace.Propagate(err, "No filepath found for test file 2 key '%v'; this is a bug in Kurtosis!", static_file_consts.TestStaticFile1ID)
```

Comment:
I think that this variable should be static_file_consts.TestStaticFile2ID

Reply:
Goood catch!